### DISCUSSION TITLE
Close issues / prs

### DISCUSSION THREAD

**Discussion by Milo123459** [community/community#3540] [Mobile] [answered]
Allow us to close issues / prs on mobile

**Comment by PranamBhat** [comment]
Hi @Milo123459 

It is going to be a new feature. Bring it here : [https://github.community/](https://github.community/)

**Comment by nplasterer** [comment]
Hi @Milo123459 you can currently close issues and prs on mobile by swiping up on the bottom triage sheet when viewing an issue or pr and then selecting close. Please let me know if you are still having issues.

**Reply by Milo123459** [reply]
I can't.. I swiped up on an issue and nothing happened

**Reply by nplasterer** [reply]
Do you mind showing me a screenshot of what you are seeing?

**Reply by Milo123459** [reply]
![image](https://user-images.githubusercontent.com/50248166/111709596-cb4bed00-883f-11eb-9c43-234ee1fe852a.png)

**Reply by candyho** [reply]
@Milo123459 Have you tried to swipe up the "Comment" area at the bottom of the screen?

**Comment by PranamBhat** [comment, marked as answer]
Hi @Milo123459 

Try swiping up from the bottom of the screen while you are in particular **issue/pr/comment**.  You can perform this action by doing the same.

**Comment by tigpt** [comment]
Hello, it used to be there a close button but now (current beta) it's not. Was It intentional ?

![IMG_9945706FB35D-1 2](https://user-images.githubusercontent.com/43767/145081692-22c762b9-14bc-406f-81ab-61c99003f0c6.jpeg)

**Reply by Iron-Ham** [reply]
Hi @tigpt 
We've moved this functionality to the top level `…` button on the issue or pull. 

![Simulator Screen Shot - iPhone Xʀ - 2021-12-07 at 13 36 57](https://user-images.githubusercontent.com/3388381/145086927-a6f34e6c-9ff8-4145-b078-155ef8eecec4.png)

**Reply by tigpt** [reply]
I've noticed it is there, but now it is harder for me to hit it. It's on the top right corner, hard to do one hand, and also sometimes (if issue is big, it gets hided and you need to scroll to the top to find it.

I understand it's impossible to please everyone, but do we need that space where it was before for something else? Can we have it in both places?

Thanks.

**Reply by bobinrinder** [reply]
@Iron-Ham I second that, it's now much harder to reach. I close lots of tickets per day and this slowed me down quite a bit. I have to scroll to the top to get to the small context menu. It's a small thing, but maybe consider having it in both places. So much space where it was before...
